pyproject.toml

```diff
@@ -0,0 +1,61 @@
+#  - git pull --rebase
+#  - Remove untracked files/dirs: git clean -fdx
+#  - maybe update version in pyperformance/__init__.py and doc/conf.py
+#  - set release date in doc/changelog.rst
+#  - git commit -a -m "prepare release x.y"
+#  - run tests: tox --parallel auto
+#  - git push
+#  - check the CI status:
+#    https://github.com/python/pyperformance/actions
+#
+# Release a new version:
+#
+#  - git tag VERSION
+#  - git push --tags
+#  - Remove untracked files/dirs: git clean -fdx
+#  - python -m build
+#  - twine upload dist/*
+#
+# After the release:
+#
+#  - set version to n+1: pyperformance/__init__.py and doc/conf.py
+#  - git commit -a -m "post-release"
+#  - git push
+
+[build-system]
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyperformance"
+dynamic = ["version"]
+license = {text = "MIT"}
+description = "Python benchmark suite"
+readme = "README.rst"
+urls = {Homepage = "https://github.com/python/pyperformance"}
+authors= [{name = "Collin Winter"}, {name= "Jeffrey Yasskin"}]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Natural Language :: English',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python',
+]
+requires-python = ">=3.7"
```

Comment:
Sadly, we've been using pyperformance with 3.6...

Reply:
https://github.com/psf/pyperf/pull/142 
pyperf does not support Python3.6 now, but I am taking a look at the support that is available from now on.
But do we need to support EOL version at pyperformance by the way?

Reply:
FYI setuptools require Python3.7 from https://setuptools.pypa.io/en/latest/history.html#v59-7-0